static/js/impala/persona_creation.js

```diff
@@ -280,17 +280,13 @@
             updatePersona();
         }).trigger('change');
 
-        // Check for native `input[type=color]` support (i.e., WebKit).
-        if ($color[0].type === 'color') {
-            $('.miniColors-trigger').hide();
-        } else {
-            $color.miniColors({
-                change: function() {
-                    $color.trigger('change');
-                    updatePersona();
-                }
-            });
-        }
+        $color.minicolors({
+            dataUris: false,
```

Comment:
This one is needed so it uses a link to the jquery.minicolors.png file (which sadly has to be in the same folder as the jquery.minicolors.css file) instead of a `data:image...` uri (which breaks if we have proper CSP in place)